.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -96,6 +89,11 @@ jobs:
           -WebAppName ${{ inputs.app-name }} `
           -SlotName ${{ inputs.destination-slot-name }}
 
+    - name: Add Azure Application Insights Release Annotation
+      uses: Lombiq/GitHub-Actions/.github/actions/add-azure-application-insights-release-annotation@dev
+      with:
+        application-insights-resource-id: ${{ inputs.application-insights-resource-id }}
+
```

Comment:
This should rather be run after the swap itself. If the ping fails, then that's bad but the app is still out in the slot, so a release annotation is due.